### DISCUSSION TITLE
PHP installation gets stuck

### DISCUSSION THREAD

**Discussion by eRez-ws** [containerbase/base#288] [Q&A] [answered]
running 
```
ARG PHP_VERSION=7.4.27
RUN install-apt php
```
gets stuck after a few minutes, with the following message:
![image](https://user-images.githubusercontent.com/35656848/152741159-9dbc055d-aa2d-4cfb-bc75-fdde75b31441.png)

(same behavior is detected on Jenkins as well)

@viceice @rarkins

**Comment by rarkins** [comment]
Can you create a minimal Dockerfile which reproduces the problem? Or add a test to this repo which fails the same way?

**Reply by eRez-ws** [reply]
1.  this is already a minimal Dockerfile, the php installation is the first actual command
2. what test do you want me to add?

**Reply by rarkins** [reply]
I do not see any Dockerfile from mobile. Are you sure there's one in this discussion or just the two lines you pasted at the start? A Dockerfile normally starts with a FROM line.

Also have you tried using install-tool?

**Comment by eRez-ws** [comment, marked as answer]
false alarm.
the root cause was a typo in the Dockerfile
